docs/growth/index.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 1
+title: Growth
+description: Get advice, support, and resources for each stage of development for your project.
+---
+
+At each stage of your project, you can get advice, support, and resources from the Flow ecosystem. Here's a breakdown of what you should be doing and what you can get at each stage:
+
+## Product Idea
+
+You've got an **idea** for an app, game, or project that you want to build on Flow.
+
+### What you should be doing
+
+* Developing the idea in a written format, such as a pitch deck, whitepaper, design doc, or business plan
+* Writing software
+* Joining a hackathon or buildathon
+
+### What you can get
+
+* Technical support
+* Product support
+* Marketing support for your launch
+* DevRel
+
+## MVP/Demo/Hackathon Project Complete
+
+You've got something that works in a place others can use it and experience key features of your product.
+
+### What you should be doing
+
+* Writing software
+* Integrating with service providers
+* Getting user feedback and improving your product
+
+### What you can get
+
+* Free credits to third-party services (e.g. Alchemy)
+* Feedback and guidance from Flow Foundation BD team
+* DevRel
+* Mentorship from startup founders
+* GrantsDAO for funding (submit project and get voted on to continue post hackathon or entirely new)
+* Instagrant Funding (for projects that are ready to go live and have signs of PMF, does not cover engineering milestones typically)
```

Comment:
I think a few of these are aspirational.

Lets update to:

Direct contact with Flow Foundation through a Developer Relations Lead.
Feedback and Guidence from the Flow Foundation team